### PR TITLE
Rename functions, classes, and modules for clarity + mjx bug fix.

### PR DESCRIPTION
**Pull request by jf514**
# __Bug Fix__
`mjx` bug found in (#66) was fixed by removing sensors from the xml files. 

# __Renaming / Name Clarification__
Also, the following were renamed:

1. `transform()` function renamed `ik_only()` to reduce confusion with standard use of "transform" to imply coordinate transformations, and to reflect that it does IK.
2. Unused `transform*.html` files were removed.
3. Many instances of the word `fit` were updated to reflect the function name change from `fit()` to `fit_offsets()` in a previous PR.
4. `STAC` class renamed to `Stac` to reflection best practices.
5. `controller.py` module renamed to `stac.py` per previous bullet, and inline with best practices.
6.  `utils.py` module renamed to `io.py` since it mostly contains io functionality. (Other functionality moved to `op_utils.py` (see below).
7. `operations.py` renamed `op_utils.py` to reflect that it mostly holds operation utilities.

### CODE REVIEW

**Review by talmo — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # stac-mjx :rat:
-Implementation of [STAC](https://ieeexplore.ieee.org/document/7030016) using [MJX](https://mujoco.readthedocs.io/en/stable/mjx.html). This is part of the VNL project. 
+`stac-mjx` is an implementation of the [Stac](https://ieeexplore.ieee.org/document/7030016) algorithm for inverse kinematics on markerless motion capture data. It's written in [MJX](https://mujoco.readthedocs.io/en/stable/mjx.html) for hardware acceleration .
```
Comment (charles-zhng) [inline README.md:2]: "It uses [JAX](https://jax.readthedocs.io/en/latest/) with [Mujoco XLA (MJX)](https://mujoco.readthedocs.io/en/stable/mjx.html) for GPU acceleration."

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # stac-mjx :rat:
-Implementation of [STAC](https://ieeexplore.ieee.org/document/7030016) using [MJX](https://mujoco.readthedocs.io/en/stable/mjx.html). This is part of the VNL project. 
+`stac-mjx` is an implementation of the [Stac](https://ieeexplore.ieee.org/document/7030016) algorithm for inverse kinematics on markerless motion capture data. It's written in [MJX](https://mujoco.readthedocs.io/en/stable/mjx.html) for hardware acceleration . 
+
+This is part of the Virtual Neurosceince Lab (VNL) project.
```
Comment (charles-zhng) [inline README.md:4]: typo (Neuroscience)

File: stac_mjx/stac.py [hunk 3]
```diff
@@ -294,10 +294,13 @@ def fit_offsets(self, kp_data):
         print(f"Standard deviation: {std}")
         return self._package_data(mjx_model, q, x, walker_body_sites, kp_data)
 
-    def transform_kps(self, kp_data, offsets):
-        """Register skeleton to keypoint data.
+    def ik_only(self, kp_data, offsets):
+        """Do only inverse kinematics (no fitting) on motion capture data.
```
Comment (charles-zhng) [inline stac_mjx/stac.py:298]: "(no marker offset optimization)"

**Review by charles-zhng — COMMENTED** [review]
Looks great--just saw some typos. Thanks for the renaming/refactoring--makes more sense!

**Pull request merged**